sklearn/cluster/_affinity_propagation.py

```diff
@@ -337,7 +351,7 @@ class AffinityPropagation(ClusterMixin, BaseEstimator):
     @_deprecate_positional_args
     def __init__(self, *, damping=.5, max_iter=200, convergence_iter=15,
                  copy=True, preference=None, affinity='euclidean',
-                 verbose=False):
+                 random_state=None, verbose=False):
```

Comment:
```suggestion
                 verbose=False, random_state=None):
```


since positional arguments are only deprecated for now, I think we still need to have new parameters at the end ATM